lib/reverse_characters.rb

```diff
@@ -1,2 +1,15 @@
 def reverse_characters(str)
+    # So, the string needs to be split into each element and returned in the opposite order.  So we should convert it to an array! 
+    # pop removes the last element of the array and returns it.  So, we just need to remove each element of the array in turn and then join them all together.
+  convertedstring = Array.new
```

Comment:
Nice work! Just for fun, what if you had begun with `convertedstring = ""`. How would your method, particularly the code inside your `times` loop change?